src/schema/v2/TargetSupply/index.ts

```diff
@@ -0,0 +1,186 @@
+import { ResolverContext } from "types/graphql"
+import {
+  GraphQLObjectType,
+  GraphQLBoolean,
+  GraphQLFieldConfig,
+  GraphQLList,
+  GraphQLString,
+} from "graphql"
+import { ArtistType } from "../artist"
+import {
+  getMicrofunnelData,
+  getTargetSupplyArtists,
+} from "../artist/targetSupply/utils/getMicrofunnelData"
+import { pageable } from "relay-cursor-paging"
+import { connectionFromArray } from "graphql-relay"
+import { artworkConnection } from "schema/v2/artwork"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { createPageCursors } from "../fields/pagination"
+
+//Example query:
+// query {
+//   targetSupply {
+//     microfunnel {
+//       metadata {
+//         highestRealized
+//       }
+//       artist {
+//         slug
+//       }
+//       artworks(first: 2) {
+// #         realizedPrice
+//         pageInfo {
+//           hasNextPage
+//           endCursor
+//         }
+//         edges {
+//           node {
+//             slug
+//           }
+//         }
+//       }
+//     }
+//   }
+// }
+
+const TargetSupplyType = new GraphQLObjectType<any, ResolverContext>({
+  name: "TargetSupply",
+  fields: {
+    microfunnel: {
+      type: new GraphQLList(
+        new GraphQLObjectType({
+          name: "TargetSupplyMicrofunnelItem",
+          fields: {
+            metadata: {
+              // TODO: share this shape with the one in ArtistTargetSupply
+              type: new GraphQLObjectType<any, ResolverContext>({
+                name: "TargetSupplyMicrofunnelItemMetadata",
+                fields: {
+                  highestRealized: {
+                    type: GraphQLString,
+                  },
+                  realized: {
+                    type: GraphQLString,
+                  },
+                  recentlySoldArtworkIDs: {
+                    type: new GraphQLList(GraphQLString),
+                  },
+                  roundedUniqueVisitors: {
+                    type: GraphQLString,
+                  },
+                  roundedViews: {
+                    type: GraphQLString,
+                  },
+                  str: {
+                    type: GraphQLString,
+                  },
+                  uniqueVisitors: {
+                    type: GraphQLString,
+                  },
+                  views: {
+                    type: GraphQLString,
+                  },
+                },
+              }),
+            },
+
+            artist: {
+              type: ArtistType,
+              resolve: async ({ slug }, {}, { artistLoader }) => {
+                const artist = await artistLoader(slug)
+                return artist
+              },
+            },
+            // TODO: what can be shared between here and ArtistTargetSupply?
+            artworksConnection: {
```

Comment:
@pepopowitz - this stuff (should) be able to be shared verbatim with the stuff that was in the other PR from last night

Reply:
Woops, ignore any recent changes that move just the resolver function to a shared location....I didn't read this comment thoroughly. I'll switch to making the whole type common.